internal/otelcollector/config/builder/config_builder_test.go

```diff
@@ -98,8 +98,21 @@ func TestMakeExtensionConfig(t *testing.T) {
 		HealthCheck: config.EndpointConfig{
 			Endpoint: "${MY_POD_IP}:13133",
 		},
+		Pprof: config.EndpointConfig{
+			Endpoint: "127.0.0.1:1777",
```

Comment:
Maybe we also expose this on `${MY_POD_IP}`? This would allow a user to open it up by adding another policy and then using tools like Parca.